### ISSUE TITLE
Error running tests for za_hansard_load_into_sayit

### ISSUE THREAD

**Issue by chrismytton** [mysociety/za-hansard#13]
```
======================================================================
ERROR: test_za_hansard_load_into_sayit (za_hansard.tests.hansard.ZAHansardSayitLoadingTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vagrant/pombola-virtualenv/local/lib/python2.7/site-packages/django/test/utils.py", line 215, in inner
    return test_func(*args, **kwargs)
  File "/vagrant/pombola-virtualenv/src/za-hansard/za_hansard/tests/hansard.py", line 180, in test_za_hansard_load_into_sayit
    call_command('za_hansard_load_into_sayit')
  File "/vagrant/pombola-virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 150, in call_command
    return klass.execute(*args, **defaults)
  File "/vagrant/pombola-virtualenv/local/lib/python2.7/site-packages/django/core/management/base.py", line 232, in execute
    output = self.handle(*args, **options)
  File "/vagrant/pombola-virtualenv/src/za-hansard/za_hansard/management/commands/za_hansard_load_into_sayit.py", line 47, in handle
    except Instance.NotFound:
AttributeError: type object 'Instance' has no attribute 'NotFound'
 
----------------------------------------------------------------------
Ran 45 tests in 20.179s
 
FAILED (errors=1)
Destroying test database for alias 'default'...
(pombola-virtualenv)vagrant@precise64:~/pombola$
```

It looks like [this line in za_hansard_load_into_sayit](https://github.com/mysociety/za-hansard/blob/master/za_hansard/management/commands/za_hansard_load_into_sayit.py#L47) is a typo and should be:

```python
except Instance.DoesNotExist:
```

**Comment by duncanparkes** [comment]
@chrismytton Is this still a problem?

**Comment by chrismytton** [comment]
@duncanparkes no idea, but if it hasn't tripped anyone else up for 2+ years I guess it's probably a non issue!

**Comment by mhl** [comment]
There were two other instances of this typo that crept in so I've fixed them too: a1666e2f448491
